src/components/Dashboard/Toolbar.vue

```diff
@@ -152,9 +153,10 @@ function toggleSelectMode() {
     </div>
 
     <v-spacer v-if="$vuetify.display.mobile" />
-    <v-pagination v-else class="flex-grow-1 overflow-hidden" v-model="currentPage" :length="pageCount" next-icon="mdi-menu-right" prev-icon="mdi-menu-left" />
+    <v-pagination v-if="showTopPagination && !$vuetify.display.mobile" class="flex-grow-1 overflow-hidden" v-model="currentPage" :length="pageCount" next-icon="mdi-menu-right" prev-icon="mdi-menu-left" />
 
-    <div class="d-flex align-center text-uppercase text-select" style="font-size: 0.8em">
+    <div class="flex-grow-1" v-if="!showTopPagination"></div>
```

Comment:
```vue
<v-pagination v-if="showTopPagination && !$vuetify.display.mobile && pageCount > 1" class="flex-grow-1 overflow-hidden" v-model="currentPage" :length="pageCount" next-icon="mdi-menu-right" prev-icon="mdi-menu-left" />
<v-spacer v-else />
```
You can reduce the number of condition branches and elements.

Reply:
Thanks this is the first time touching something Vue for me.
So the v-spacer/if/else part doesn't come naturally to me. 😂